docs/docs/spec/metrics/metrics-examples.md

```diff
@@ -16,13 +16,13 @@ Some metrics and endpoints are very straightforward, but having a reference for
 ## Prometheus Metrics
 
 ### Economics metrics
-* `eigen_fees_earned_total{token="ETH"}`
-* `eigen_slashing_incurred_total{token="ETH"}`
-* `eigen_total_balance_total{token="ETH"}`
+* `eigen_fees_earned_total{token="ETH", unit="wei", strategy="eigenDA"}`
+* `eigen_slashing_status{avs="eigenDA"}`
```

Comment:
remove eigenDA. see previous comment.

Reply:
Can we also add a note that there is no slashing for the upcoming testnet launch, so this should be optional.

Reply:
You expressed a desire for an `avs` label for the `eigen_slashing_status`. If we keep it, can you please suggest which value to give to this label?

Got it for the note.

Reply:
Sorry for the confusion. I might have misread here. Keep eigenDA for the avs label, but remove it for all strategy labels, since there it sounds like it's talking about an eigenDA token.

Reply:
Done [d0e911](https://github.com/NethermindEth/el-node-spec/pull/58/commits/3d0e911b83f03ccb7740f16dddcf4ed43ee53e72)